presentation/src/main/java/me/androidbox/presentation/navigation/NavigationGraph.kt

```diff
@@ -140,7 +141,10 @@ fun NavigationGraph(
                 defaultValue = null
             }, navArgument(MENU_ACTION_TYPE) {
                     type = NavType.StringType
-                })
+                }),
+            deepLinks = listOf(navDeepLink {
+                this.uriPattern = "https://androidbox.me/event{eventId}/{menuActionType}"
```

Comment:
Is this correct I need to add both the eventId, and menuActionType. The menuActionType was used as part of the dropdownmenu for open/edit/delete.

Reply:
Another question do I need to append anything to the route: `route = Screen.EventScreen.route`

Reply:
@MKiperszmid Updated to the following to pass in a default OPEN

```
navArgument(MENU_ACTION_TYPE) {
                type = NavType.StringType
                defaultValue = AgendaMenuActionType.OPEN
                })
```